replit.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs }: {
+	deps = [
+	pkgs.nodejs-18_x
+    pkgs.nodePackages.typescript-language-server
+    pkgs.nodePackages.pnpm
+	];
+}
```

Comment:
is this correct formatting? looks weird

Reply:
Fixed indentation 
[2307ab7](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/248/commits/2307ab7806247b9ee7eac12602f82556b63042f5)

Reply:
Fixed indentation
[7abb935](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/248/commits/7abb935d659eb64685437cf3cf32573592f98537)